Gem/Transfer/Scenes/AmountScene.swift

```diff
@@ -121,19 +123,21 @@ struct AmountScene: View {
                                 )
                             } secondary: {
                                 AnyView(
-                                    Button(Localized.Transfer.max, action: useMax)
-                                        .buttonStyle(ColorButton.lightGray(paddingHorizontal: 16, paddingVertical: 8))
-                                        .fixedSize()
+                                    HStack {
+                                        Button(Localized.Transfer.max, action: useMax)
+                                            .buttonStyle(ColorButton.lightGray(paddingHorizontal: 16, paddingVertical: 8))
+                                            .fixedSize()
+                                    }
                                 )
                             }
                         }
-                        .frame(maxWidth: Spacing.scene.button.maxWidth)
+                        .frame(maxWidth: .infinity)
```

Comment:
need to make sure button is not larger than `Spacing.scene.button.maxWidth`

Reply:
yeah. Button still fixedSize and haven't changed. The frame changed for container view for iPad to fix ipad layout for that button